src/fiatconnect/index.ts

```diff
@@ -47,15 +47,35 @@ function convertToFiatConnectCryptoCurrency(cicoCurrency: CiCoCurrency): CryptoT
   }[cicoCurrency]
 }
 
+interface LinkAccountScreenTranslationKeys {
+  bodyTitle: string
+  description: string
+  header: string
+}
+
 /**
  * Small helper function to map the account type to its name used in the base translation
  * file.
  */
-export function convertAccountTypeToTranslation(accountType: FiatAccountType): string {
+export function accountTypeToLinkAccountTranslations(
```

Comment:
nit: I think this function would probably be better placed in the LinkAccountScreen.tsx since its use is so closely tied to that component. 

Nit: also if you do move it to that file you can probably make the function name something less verbose like `getTranslationStrings`